go.mod

```diff
@@ -4,27 +4,28 @@ go 1.13
 
 require (
```

Comment:
Could you leave the upgrades to a separate PR?

Reply:
Sure, will do since that will be more concise.